portal/models/research_study.py

```diff
@@ -42,3 +42,21 @@ def update_from_fhir(self, data):
         if 'status' in data:
             self.status = data.get('status')
         return self
+
+
+def add_static_research_studies():
+    """Seed database with default static research studies
+
+    Idempotent - run anytime to pick up any new relationships in existing dbs
+
+    """
+    base = {
+      "id": 0,
+      "title": "Base Study",
```

Comment:
From an i18n angle, will these titles be viewable by staff or patients?

ie do we need to add them to the [existing static string extraction code](https://github.com/uwcirg/truenth-portal/blob/develop/portal/models/i18n.py#L23-L37)

Reply:
ugh - yes, they may, as part of TN-2634.